crates/context_aware_config/src/api/config/types.rs

```diff
@@ -1,16 +1,17 @@
-use serde::Serialize;
+use serde::{Deserialize, Serialize};
 use serde_json::{Map, Value};
 
-#[derive(Serialize)]
+#[derive(Serialize, Deserialize)]
 pub struct Config {
     pub contexts: Vec<Context>,
     pub overrides: Map<String, Value>,
     pub default_configs: Map<String, Value>,
 }
 
-#[derive(Serialize, Clone)]
+#[derive(Serialize, Clone, Deserialize)]
 pub struct Context {
     pub id: String,
     pub condition: Value,
+    pub priority: i32,
```

Comment:
Are okay with serving `priority` in the `getConfig` response?

Reply:
We can, but this shouldn't break the API contract we have with clients @pratikmishra356

Reply:
yes , its not breaking api contract with clients , since we are adding a field